packages/block-editor/src/components/block-tools/style.scss

```diff
@@ -119,6 +118,226 @@
 			border-right-color: $gray-200;
 		}
 	}
+
+	// on desktop and tablet viewports the toolbar is fixed
+	// on top of interface header
+	@include break-medium() {
+		&.is-fixed {
+
+			// position on top of interface header
+			position: fixed;
+			top: $grid-unit-50 - 2;
+			// leave room for block inserter
+			left: $grid-unit-80 + $grid-unit-40;
+			// Don't fill up when empty
+			min-height: initial;
+			// remove the border
+			border-bottom: none;
+			// has to be flex for collapse button to fit
+			display: flex;
+
+			&.is-collapsed {
+				left: $grid-unit-10 * 32;
+			}
+
+			.is-fullscreen-mode & {
+				top: $grid-unit - 2;
+				// leave room for block inserter
+				left: $grid-unit-80 + $grid-unit-70;
+				&.is-collapsed {
+					left: $grid-unit-10 * 35;
+				}
+			}
+
+			& > .block-editor-block-toolbar__group-collapse-fixed-toolbar {
+				border: none;
+
+				// Add a border as separator in the block toolbar.
+				&::after {
+					content: "";
+					width: $border-width;
+					height: 24px;
+					margin-top: $grid-unit + $grid-unit-05;
+					margin-bottom: $grid-unit + $grid-unit-05;
+					background-color: $gray-300;
+					position: absolute;
+					left: 44px;
```

Comment:
This is probably the sketchiest value of them all, maybe it needs a comment? Is it icon width + icon padding?